ocp_resources/node_health_check.py

```diff
@@ -0,0 +1,10 @@
+from ocp_resources.resource import NamespacedResource
+
+
+class NodeHealthCheck(NamespacedResource):
+    """
+    NodeHealthCheck object.
+    Reference : https://docs.openshift.com/container-platform/4.10/nodes/nodes/eco-node-health-check-operator.html
```

Comment:
Please replace with a reference to the project (and not a 4.10 doc )

Reply:
Done. Added link to the GitHub repo